### PR TITLE
build of imagemagick should depend on xorg only if building with x11 …

### PR DESCRIPTION
**Pull request by marlukcz**
build of imagemagick should depend on xorg only if building with x11

### CODE REVIEW

**Comment by marlukcz** [comment]
Anyone help ?

**Comment by marlukcz** [comment]
help wanted

**Comment by iMichka** [comment]
Hi. Yes, sorry. I hope I can come back to this and help you tomorrow.

This will need to be rebased too.

File: Formula/imagemagick.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@ class Imagemagick < Formula
   option "without-modules", "Disable support for dynamically loadable modules"
   option "without-threads", "Disable threads support"
   option "with-zero-configuration", "Disables depending on XML configuration files"
+  option "without-x", "Compile without xorg as default"
```
Comment (iMichka) [inline Formula/imagemagick.rb]: Please safeguard this with an `unless OS.mac?` statement, as this is a Linux only option.

**Pull request closed without merging**